js/script.js

```diff
@@ -0,0 +1,74 @@
+$(document).ready(function(){
+
+$('.carousel').carousel({
+        interval: 3000
+    })
```

Comment:
Since the carousel is initially empty, you don't have to initialize the carousel. You'll only have to after adding the images.